ega-data-api-filedatabase/src/main/java/eu/elixir/ega/ebi/downloader/service/internal/LocalEGAFileServiceImpl.java

```diff
@@ -0,0 +1,109 @@
+
```

Comment:
this should not be here, it was removed in: https://github.com/neicnordic/ega-data-api/pull/3